src/encoder.rs

```diff
@@ -832,8 +833,8 @@ impl<T: Pixel> FrameInvariants<T> {
     fi
   }
 
-  pub fn set_ref_frame_sign_bias(&mut self) {
-    for i in 0..INTER_REFS_PER_FRAME {
+  pub fn set_ref_frame_sign_bias(&mut self, inter_cfg: &InterConfig) {
+    for i in 0..inter_cfg.inter_refs_per_frame() {
```

Comment:
I don't think the optimization is really worth it here?

Reply:
but it clearly corrected the looping thru unnecessary iterations.

Reply:
So the thing that bothers me is that this is spec designed behavior to propagate reference frame context. I think it's OK to skip in this case, but it's a different thing than not skipping searching the reference frames.